Stratis.Bitcoin.Features.BlockStore/LoopSteps/ReorganiseBlockRepositoryStep.cs

```diff
@@ -41,14 +41,12 @@ internal ReorganiseBlockRepositoryStep(BlockStoreLoop blockStoreLoop, ILoggerFac
         /// <inheritdoc/>
         internal override async Task<StepResult> ExecuteAsync(ChainedBlock nextChainedBlock, CancellationToken cancellationToken, bool disposeMode)
         {
-            this.logger.LogTrace("({0}:'{1}/{2}',{3}:{4})", nameof(nextChainedBlock), nextChainedBlock.HashBlock, nextChainedBlock.Height, nameof(disposeMode), disposeMode);
-
```

Comment:
same here as above

Reply:
Same comment as for CheckExistsInPending step...